datadog/resource_datadog_powerpack.go

```diff
@@ -654,24 +716,41 @@ func ppkWidgetsToDashboardWidgets(ppkWidgets []datadogV2.PowerpackInnerWidgets)
 		if widgetDefinition == nil {
 			continue
 		}
+		widgetDefinition = normalizeTerraformWidgetDef(widgetDefinition)
 		// Add new powerpack-supported widgets here
 		// We save Powerpack widgets as Dashboard widgets so we need to convert them to the appropriate widget definition object.
 		widgetType := widgetDefinition["type"]
 		switch widgetType {
 		case "alert_graph":
 			definition = datadogV1.AlertGraphWidgetDefinitionAsWidgetDefinition(buildDatadogAlertGraphDefinition(widgetDefinition))
+		case "alert_value":
+			definition = datadogV1.AlertValueWidgetDefinitionAsWidgetDefinition(buildDatadogAlertValueDefinition(widgetDefinition))
+		case "change":
+			definition = datadogV1.ChangeWidgetDefinitionAsWidgetDefinition(buildDatadogChangeDefinition(widgetDefinition))
 		case "check_status":
 			definition = datadogV1.CheckStatusWidgetDefinitionAsWidgetDefinition(buildDatadogCheckStatusDefinition(widgetDefinition))
+		case "event_stream":
+			definition = datadogV1.EventStreamWidgetDefinitionAsWidgetDefinition(buildDatadogEventStreamDefinition(widgetDefinition))
+		case "event_timeline":
+			definition = datadogV1.EventTimelineWidgetDefinitionAsWidgetDefinition(buildDatadogEventTimelineDefinition(widgetDefinition))
 		case "free_text":
 			definition = datadogV1.FreeTextWidgetDefinitionAsWidgetDefinition(buildDatadogFreeTextDefinition(widgetDefinition))
 		case "iframe":
 			definition = datadogV1.IFrameWidgetDefinitionAsWidgetDefinition(buildDatadogIframeDefinition(widgetDefinition))
 		case "image":
 			definition = datadogV1.ImageWidgetDefinitionAsWidgetDefinition(buildDatadogImageDefinition(widgetDefinition))
+		case "manage_status":
```

Comment:
**thought(out-of-scope):** I wonder if we should consider using [the constants from datadogV1](https://github.com/DataDog/datadog-api-client-go/blob/aefcec2d6172891dbba35821a6bb96958f3a8aa7/api/datadogV1/model_monitor_summary_widget_definition_type.go#L18) in this switch statement.

For example, here it would probably make this code a little bit easier to visually check `MONITORSUMMARYWIDGETDEFINITIONTYPE_MANAGE_STATUS` matching:
- `MonitorSummaryWidgetDefinitionAsWidgetDefinition` and,
- `buildDatadogManageStatusDefinition`

Also for others where the string isn't immediately obvious but the constant likely is (added bonus: using constants from our Go client is less to worry about hah)

Reply:
Oo I did not realize we have constants available for widgets, what do you think about be adding this in a follow up?

Reply:
Yep yep, that sounds good as its own follow-up PR (it may turn out to be a bad idea after we dig into it)